include/smack/SmackRep.h

```diff
@@ -150,6 +166,14 @@ class SmackRep {
   virtual const Stmt* alloca(llvm::AllocaInst& i);
   virtual const Stmt* memcpy(const llvm::MemCpyInst& msi);
   virtual const Stmt* memset(const llvm::MemSetInst& msi);
+  virtual const Stmt* loadAsBytes(const llvm::LoadInst& li);
+  virtual const Stmt* storeAsBytes(const llvm::StoreInst& si);
```

Comment:
Wouldn't it be cleaner to implement the following instead?
````
virtual const Stmt* load(const llvm::Value* addr, const llvm::Value* val);
virtual const Stmt* store(const llvm::Value* addr, const llvm::Value* val);
````
Then case splitting on `SmackOptions::BitVectors`, region calculation, etc., can be done once inside, rather than spread out over each invocation.